CHANGES.md

```diff
@@ -95,6 +95,10 @@ config option `[scheduling]stop after cycle point`.
 
 ### Fixes
 
+[#3919](https://github.com/cylc/cylc-flow/pull/3919) - Fixed a bug in which
+remote tests failed when a newline character was included in the output from
+sed.
+
```

Comment:
Probably best to drop the changelog commit as this is just a test fix

Reply:
Fine with me. I'll remove it.